src/mac/pxWindowNative.mm

```diff
@@ -1067,6 +1067,8 @@ -(void)drawRect:(NSRect)dirtyRect {
 pxError pxWindow::term()
 {
   NSWindow* window = (NSWindow*)mWindow;
+  WinDelegate* delegate = (WinDelegate*)mDelegate;
+  [delegate release];
```

Comment:
We should set the window's delegate to nil before releasing mDelegate ([window setDelegate:nil];).  We should also set mDelegate to nil after this.